histogram_analysis.py

```diff
@@ -4,155 +4,74 @@
 from bokeh.charts import Bar, output_file, show
```

Comment:
Overall: Solid work incorporating the comments from the MP2 feedback. Everything is much cleaner now and easier to read :+1: 

Documentation is pretty good as well. 

Keep it up! :high_heel: :cactus: :snake: :sagittarius: :airplane: :package: